.circleci/config.yml

```diff
@@ -23,27 +23,32 @@ commands:
       - run:
           name: Generate and apply migration
           command: npx prisma migrate dev
-  setup_docker_compose:
+  setup_docker_compose_dependencies:
     steps:
       - setup_remote_docker:
           version: 20.10.14
       - run:
           name: Run images of services declared in docker-compose.yml
-          command: docker-compose up -d
-  create_test_env:
+          command: docker-compose --profile dependencies up -d
+  create_env:
     steps:
       - run:
-          name: Create test.env file
+          name: Create .env file
           command: |
-            echo 'DATABASE_URL="postgresql://postgres:postgres@localhost:5432/postgres"' > test.env
-            echo 'POSTGRES_USER="test_postgres"' >> test.env
-            echo 'POSTGRES_PASSWORD="test_postgres"' >> test.env
-            echo 'PGADMIN_EMAIL="test@email.com"' >> test.env
-            echo 'PGADMIN_PASSWORD="test_password"' >> test.env
-            echo 'JWT_SECRET="testJWTValue4590823HadToAddIt"' >> test.env
-            echo 'BCRYPT_SALT="10"' >> test.env
-            echo 'REDIS_PASSWORD="red_test_password"' >> test.env
-            echo 'REDIS_URL="redis://localhost:6379"' >> test.env
+            cat > .env \<< EOF
+            DATABASE_URL="$DATABASE_URL"
+            POSTGRES_USER="$POSTGRES_USER"
+            POSTGRES_PASSWORD="$POSTGRES_PASSWORD"
+            PGADMIN_EMAIL="$PGADMIN_EMAIL"
+            PGADMIN_PASSWORD="$PGADMIN_PASSWORD"
+            JWT_SECRET="$JWT_SECRET"
+            BCRYPT_SALT="$BCRYPT_SALT"
+            REDIS_PASSWORD="$REDIS_PASSWORD"
+            REDIS_URL="$REDIS_URL"
+            EOF
+            cat .env > test.env
```

Comment:
Not sure why you need to create both `.env` and `test.env`? I thought we only need to create `test.env` for `jest` tests, and for normal builds we could fetch envs from CI? (might be mistaken though)

Reply:
That's because later, when I run `setup_docker_compose_dependencies` to setup db, docker look sat .env file I think.

Reply:
ok, if that's the case then fine 👍🏻 Would be good to be sure of that though 🙂